docs/cloud/cloud.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+label: Cloud
+--- 
+
+
+## Getting Started with Dragonfly Cloud
+
+A Dragonfly Cloud data store represents a Redis Protocol (RESP) endpoint. To create a data store, on the Data Store tab click +Data Store. The minimum configuration consists of a name, cloud provider, cloud region and plan. Note that the cloud provider can not be modified once the data store is created.
```

Comment:
please format better with line breaks please

Reply:
Can you use an image for +Data Store 
![image](https://github.com/dragonflydb/documentation/assets/97001/bcb0d2bc-172a-4441-9825-c36eb1a267c9)